pub-dev.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Reminder that this isn't the most portable hashbang to use for Bash.

`#!/usr/bin/env bash` is.